anchor-reference-server/src/main/java/org/stellar/anchor/reference/service/CustomerService.java

```diff
@@ -250,6 +259,14 @@ public CustomerField createBankAccountNumberField(String type) {
     return field;
   }
 
+  public CustomerField createBankAccountTypeField(String type) {
+    CustomerField field = new CustomerField();
+    field.setType("string");
+    field.setDescription("bank account type of the customer");
+    field.setOptional(!type.equals(Customer.Type.SEP31_RECEIVER.toString()));
+    return field;
+  }
```

Comment:
Since `bank_account_type` is an enum, lets add the `choices` list that can be defined for fields with a limited set of accepted values.

Reply:
Added